internal/generate/clusterserviceversion/clusterserviceversion_test.go

```diff
@@ -149,7 +155,28 @@ var _ = Describe("Generating a ClusterServiceVersion", func() {
 					WithBase(csvBasesDir, goAPIsDir, projutil.InteractiveHardOff),
 					WithBaseWriter(tmp),
 				}
-				Expect(g.Generate(cfg, opts...)).ToNot(HaveOccurred())
+				Expect(g.Generate(opts...)).ToNot(HaveOccurred())
+				outputFile := filepath.Join(tmp, "bases", makeCSVFileName(operatorName))
+				outputCSV, _, err := getCSVFromFile(outputFile)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(outputFile).To(BeAnExistingFile())
+
+				annotations := outputCSV.GetAnnotations()
+				Expect(annotations).ToNot(BeNil())
+				Expect(annotations).Should(HaveKey(metricsannotations.BuilderObjectAnnotation))
+				Expect(annotations).Should(HaveKey(metricsannotations.LayoutObjectAnnotation))
+			})
+			It("should have sdk labels in annotations when generating with config", func() {
```

Comment:
This test path is already tested, should a new test be added to also check with `WithBaseCreator`